lib/admin_api.ts

```diff
@@ -0,0 +1,66 @@
+/**
+ * Main entrypoint for the Administration APIs handlers
+ */
+import { IContext } from "azure-function-express";
+
+import * as winston from "winston";
+
+import { setAppContext } from "./utils/middlewares/context_middleware";
+
+import { configureAzureContextTransport } from "./utils/logging";
+
+import { DocumentClient as DocumentDBClient } from "documentdb";
+
+import * as documentDbUtils from "./utils/documentdb";
+
+import { createAzureFunctionHandler } from "azure-function-express";
+
+import { ServiceModel } from "./models/service";
+
+import { CreateService, UpdateService } from "./controllers/admin_services";
+
+import { GetDebug } from "./controllers/debug";
+
+import { getSecureExpressApp } from "./utils/express";
+
+// Setup Express
+const app = getSecureExpressApp();
```

Comment:
you should create an `express` instance in here and pass it to this method to configure it with the security middlewares

Reply:
why ?

Reply:
it's a better design - because that's the minimum information needed by `getSecureExpressApp` function to be useful, your intent was to refactor the security configurations into a common function, not the instantiation of an express app - always go for the minimal dependency / responsibility